dev-server/documents/html/configurations.mustache

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
```

Comment:
This template has a rather generic name (`configurations.mustache`). I suggest something like `ignore-other-configuration.mustache` would leave room for other configuration-related tests in future.

Reply:
Sure.